tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -152,9 +155,24 @@ oe_result_t gen_report(oe_enclave_t* enclave)
                 NULL, remote_report, report_size, &parsed_report);
             if (result != OE_OK)
             {
-                log("Failed to verify report. result=%u (%s)\n",
+                printf(
+                    "Failed to verify report. result=%u (%s)\n",
                     result,
                     oe_result_str(result));
+
+                // Print TCB Info to console if verification failed
+                printf("\nQEID: ");
```

Comment:
nit: Prefix the value with `0x`

Reply:
use oe_hex_dump

---
In reply to: [395719207](https://github.com/openenclave/openenclave/pull/2708#discussion_r395719207) [](ancestors = 395719207)